### ISSUE TITLE
Player View

### ISSUE THREAD

**Issue by SpaceDiceMan** [Stefouch/myz-zonemap#5]
* Please make the grid-cells transparant in Player View (so that the players can identify (and admire) the map underneath). Explored cells can be indicated by either icons or a colored border instead.
* Also please allow the rot icon to also show in Player View (since players will know the Rot value after exploring), this allows players to pick routes. [I see that the red border of high rot is already visible in player view [please keep]].

**Comment by Stefouch** [comment]
I agree for the rot level, as it is very easy for the players to determine it.

But I'm unsure to make the "fog-of-war" transparent. I like when my players have to discover the map sector by sector.
In my games, my players have a printed map with no data. And when they ask for information, I saw them the Player View from here.

**Comment by SpaceDiceMan** [comment]
Ah, I wasnt clear. 
The player map covers the background (printed) map because the grid cells are opaque. If the grid cells would not be opaque you could see the background. The information [name, rot etc] visible is fine.

**Comment by Stefouch** [comment]
Actually, I think it's better to have an opaque fog-of-war.

- It allows the GM to use a detailed map background image, without revealing too much to the players. For example, if you take a look at the NYC map in the corebook, there is a drawing depicting Rikers Island. Maybe you don't want your players to see this information.

- I like the idea of a map of the Zone being revealed sector by sector. After several sectors explored, a wider image starts to appear. It feels like the Ark conquered some territories, which is more or less true.

- Being surrounded by a fog-of-war seems oppressive at first. The Ark feels alone in the dark. I like this feeling for my players.

- The players see immediately where are unexplored sectors.

Nonetheless, I'll implement a button to "reveal an unexplored" sector. Sometimes, a NPC might give to the players the location of a special Zone sector.

**Comment by bjornhagstrom** [comment]
In player view the background map is not visible at all for me (MacOS Safari/Chrome). I disapears when I enter player mode. It would be great to have it visible in both views (maybe a check box for those that do not want players to se it all?).

Thanks for the great work, it is a great resource!

PS Is this project still being maintained?
